picky/src/ssh/mod.rs

```diff
@@ -0,0 +1,35 @@
+use base64::read::DecoderReader;
+use base64::write::EncoderWriter;
+use byteorder::ReadBytesExt;
+use std::io::{self, Read};
```

Comment:
Please, move these `use`s after the `pub mod` directives (that's how we do at the other places too).